src/wallets/WalletOverview.tsx

```diff
@@ -9,12 +11,19 @@ import './WalletOverview.scss'
 import {isDefinedNetworkName} from '../config/type'
 
 interface WalletOverviewProps {
-  availableBalance: BigNumber
+  availableBalance: Option<BigNumber>
 }
 
 export const WalletOverview = ({availableBalance}: WalletOverviewProps): JSX.Element => {
   const {networkName} = BackendState.useContainer()
-  const etcAvailableBalance = availableBalance.shiftedBy(-ETC_CHAIN.decimals).toNumber()
+
+  const etcAvailableBalance = pipe(
+    availableBalance,
+    fold(
+      () => 0,
```

Comment:
I think we should display a message instead of zero in case the balance is unavailable.
The values we display should be factual.

Reply:
Do you have any suggestion what message to display there in case of error? So far, I've hidden the balance.